paddle/fluid/operators/crop_tensor_op.cc

```diff
@@ -0,0 +1,269 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
2018 -> 2019

Reply:
Done. 
- add python API test in test_layers.py
- modified `Copyright (c) 2018` to `Copyright (c) 2019` of 4 files: crop_tensor_op.cc、crop_tensor_op.cu、crop_tensor_op.h、test_crop_tensor_op.py